enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -191,4 +201,30 @@ impl<'a> Handler<'a> {
             ret
         }
     }
+
+    /// Do an arch_prctl() syscall
+    pub fn arch_prctl(&mut self) -> u64 {
+        // TODO: Check that addr in %rdx does not point to an unmapped address
+        // and is not outside of the process address space.
+
+        match ArchPrctlTask::from(self.aex.gpr.rsi) {
+            ArchPrctlTask::ArchSetFs => {
+                self.aex.gpr.fsbase = self.aex.gpr.rdx;
+                0
+            }
+
+            // TODO: Fix me
+            ArchPrctlTask::ArchGetFs => ErrNo::EINVAL.into_syscall(),
```

Comment:
```suggestion
            ArchPrctlTask::ArchGetFs => ErrNo::ENOSYS.into_syscall(),
```

The value is valid, just not implemented.